src/pages/public_registration/log/LogPanel.tsx

```diff
@@ -14,9 +14,10 @@ import { LogEntryItem } from './LogEntryItem';
 interface LogPanelProps {
   registration: Registration;
   tickets: Ticket[];
+  refetchData?: () => Promise<void>;
```

Comment:
Ser at `refetchData` prop-drilles ganske mange komponeter nå for å bare brukes den ene plassen helt nederst. Når man ser det blir sånn er det lurt å vurdere om man heller skal lage en Context så man bare trenger å gjøre endringer i de komponentene som faktisk skal bruke den.
Nå ser jeg at det i dette tilfellet er basert på den gamle løsningen vi har med prop-drilling av `refetchRegistrationAndTickets`, så det er for si vidt greit nok. Ikke noe vi trenger å endre nå, men noe som er viktig å tenke på da, for det blir bare verre 😅